### ISSUE TITLE
List bullet icon is not visible when colour preferences are changed in Firefox

### ISSUE THREAD

**Issue by pharingee** [bbc/simorgh#4945]
Describe the bug
The bullet icon in psammead-bulleted-list is not visible when colour preferences are changed in Firefox

To Reproduce

1. Using Firefox, change colour preferences with instructions [here](https://support.mozilla.org/en-US/kb/change-fonts-and-colors-websites-use)
2. Go to https://www.test.bbc.com/pidgin/23248703
3. See error

**Expected behavior**
The bullet icon should be visible when colour preferences are changed in Firefox
**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**
<img width="828" alt="Screenshot 2019-12-11 at 11 21 40" src="https://user-images.githubusercontent.com/22519682/70989438-8d4f9e80-20bb-11ea-8709-f2baae55c78b.png">

- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by ryanmccombe** [comment]
Example settings to cause it:

![Screen Shot 2019-12-17 at 12 12 56](https://user-images.githubusercontent.com/8601460/70994668-a3169100-20c6-11ea-9846-5bda98820c92.png)

Agreed to fix this for launch

May require some investigation, but likely to be a CSS addition

**Comment by ryanmccombe** [comment]
Investigated this, but unassigning myself as no prospect that I will get this fixed before going on holiday, and the increased estimate may affect prioritisation.

Fixing this would require a rethink of how we have implemented these bulleted lists.

Bulleted lists use background images for the bullets - this is going to be problematic when it comes to high contrast modes.  They tend to be not that compatible with background images - can't find any official docs on this, but plenty of people claiming it - this seems to be the most authoritive:

https://engineering.khanacademy.org/posts/windows-high-contrast-mode.htm

> Background colors and background images are removed.

> **Key takeaway**
> Only use background-color and background-image CSS for decorative styling. Windows High Contrast Mode considers them unnecessary to understand the page.

**Comment by gavinspence** [comment]
Won't address this issue as it will require a re-implementation of bulleted lists.
